src/Makefile.am

```diff
@@ -2,13 +2,13 @@
 EXTRA_DIST = \
 	.indent.pro
 
-ubindir = ${prefix}/bin
-usbindir = ${prefix}/sbin
```

Comment:
With the current state of MR, the same exec_prefix will be used for `ubindir` && `bindir` 
* https://www.gnu.org/software/automake/manual/html_node/Standard-Directory-Variables.html
So with the current state of MR  `ubindir` == `bindir` and  `usbindir` == `sbindir`.

Do we keep`ubindir` and `usbindir` in such case?
Or they are intentionally separate in Makefile.am so one can use make variables to have different  values for `ubindir` and `bindir` e.g.
```
make install ubindir=/tmp/bin usbindir=/tmp/sbin
```

Otherwise LGTM and +1 for removing hacks :-)

Reply:
I want to remove them, but in a separate PR.  I'd say even in a separate release, just to make sure we don't break anything at some point.

Reply:
Yes, I want that; however, I'd like the transition to be gradual, to make sure each patch is easy to review for correctness.